### PR TITLE
online_status: fix timeout for ping

### PR DESCRIPTION
**Pull request by lasers**
Reviewing a new module #1163 has been brought to my attention that we're not applying the `timeout` config to the `ping` section. This PR will include the `-W` option for `self.timeout`. Tested OK in `python2` and `python3`.

Like `ping`, I elicted no response from contributers and/or owners for days. My `timeout`, in days, have been reached so I will stop contributing until further notice. We can try again when everybody is not too occupied. Have a good day. :-)

### CODE REVIEW

File: py3status/modules/online_status.py [hunk 1]
```diff
@@ -66,22 +66,21 @@ class Meta:
     def post_config_hook(self):
         self.color_on = self.py3.COLOR_ON or self.py3.COLOR_GOOD
         self.color_off = self.py3.COLOR_OFF or self.py3.COLOR_BAD
+        self.ping_cmd = ['ping', '-c', '1', '-W', str(self.timeout), self.url]
```
Comment (tobes) [inline py3status/modules/online_status.py]: `str(self.timeout)` did you test this with python2?

Comment (lasers) [reply py3status/modules/online_status.py]: Yes. It looks okay.
```
1d [chris:~/src/py3status/py3status/modules] online_status-timeout(+2/-1)+ 130 ± python3 online_status.py
Module `test_module`: ['ping', '-c', '1', '-W', '2', 'google.com']
{'full_text': '●', 'color': '#00FF00', 'cached_until': 1511399543.0}
Module `test_module`: ['ping', '-c', '1', '-W', '2', 'google.com']
{'full_text': '●', 'color': '#00FF00', 'cached_until': 1511399544.0}
^C%
1d [chris:~/src/py3status/py3status/modules] online_status-timeout(+2/-1)+ ± python2 online_status.py
Module `test_module`: ['ping', '-c', '1', '-W', '2', 'google.com']
{'color': '#00FF00', 'full_text': u'\u25cf', 'cached_until': 1511399546.0}
Module `test_module`: ['ping', '-c', '1', '-W', '2', 'google.com']
{'color': '#00FF00', 'full_text': u'\u25cf', 'cached_until': 1511399547.0}
^C%
```

Comment (lasers) [reply py3status/modules/online_status.py]: Switch to `%s` anyway?

Comment (tobes) [reply py3status/modules/online_status.py]: > `1d [chris:~/src/py3status/py3status/modules] online_status-timeout(+2/-1)+ ± python2 online_status.py`

for these changing the prompt to `$` 'user' or `#` 'root' is customary as it makes things more readable and doesn't leak information so 
```
$ python2 online_status.py
...
```

not a huge deal

**Comment by tobes** [comment]
Nice work, I'm going to add this into 3.8 as soon as we get 3.7 out,  I don't see this causing issues but I'm catious.  Also partly a reminder that this module needs some love.  But we'll merge this PR first before worrying about the love needed.

**Comment by tobes** [comment]
@lasers thanks mate

**Pull request merged**